src/core/QCheck.ml

```diff
@@ -775,6 +750,11 @@ module Shrink = struct
     | None -> ()
     | Some shrink -> list_elems shrink l yield
 
+  let string (*?(shrink=char)*) s yield =
```

Comment:
What's up with the commented parameter?

Reply:
This is a left-over. I think I included it initially and then commented it out at some point. I just removed it in 4aad609.
We could consider adjusting the `Shrink.string` signature to include it, but we should do so in a separate PR.